### PR TITLE
fix(tree): add `tree.data.collapsed`

### PR DESCRIPTION
**Pull request by susiwen8**

### CODE REVIEW

File: en/option/series/tree.md [hunk 1]
```diff
@@ -294,6 +294,10 @@ The name of the tree node, used to identify each node.
 
 The value of the node, displayed in the tooltip.
 
+### collapsed(boolean)
+
+Whether collpase node at initialization。
```
Comment (plainheart) [inline en/option/series/tree.md:299]: - A typo of `collapse` here. 
- `to` is missing after `Whether`.

Comment (susiwen8) [reply en/option/series/tree.md:299]: Great catch!

File: en/option/series/tree.md [hunk 2]
```diff
@@ -202,7 +202,6 @@ Describes the style of the text label corresponding to the leaf node.
     defaultShowLabel = true,
     formatter1d = true
 ) }}
-
```
Comment (plainheart) [inline en/option/series/tree.md:205]: This empty line should be reserved for a pretty format.

File: en/option/series/tree.md [hunk 3]
```diff
@@ -294,6 +293,10 @@ The name of the tree node, used to identify each node.
 
 The value of the node, displayed in the tooltip.
 
+### collapsed(boolean)
+
+Whether to collpase node at initialization.
```
Comment (plainheart) [inline en/option/series/tree.md:298]: Typo of `collapse` doesn't get fixed. `collpase` -> `collapse`.

**Review by plainheart — APPROVED** [review]

**Pull request merged**